crates/bevy_ecs/src/schedule/executor/multi_threaded.rs

```diff
@@ -283,7 +281,7 @@ impl<'scope, 'env: 'scope, 'sys> Context<'scope, 'env, 'sys> {
             .push(SystemResult { system_index })
             .unwrap_or_else(|error| unreachable!("{}", error));
         if let Err(payload) = res {
-            eprintln!("Encountered a panic in system `{}`!", &*system.name());
+            log::error!("Encountered a panic in system `{}`!", &*system.name());
```

Comment:
there's no guarantee a logger is set here and this error is too important to risk to hide, it should keep using eprintln